tgui/packages/tgui/interfaces/Stack.js

```diff
@@ -0,0 +1,177 @@
+import { useBackend } from "../backend";
+import { Box, Button, Section, Collapsible, Table } from "../components";
+import { Window } from "../layouts";
+
+export const Stack = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    amount,
+    recipes,
+  } = data;
+
+  const height = Math.max(92 + Object.keys(recipes).length * 21.8, 250);
+
+  return (
+    <Window
+      width={400}
+      height={Math.min(height, 500)}
+      resizable>
+      <Window.Content scrollable>
+        <Section title={"Amount: " + amount}>
+          <RecipeList recipes={recipes} />
+        </Section>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const RecipeList = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    recipes,
+  } = props;
+
+  let sortedKeys = Object.keys(recipes).sort((a, b) => {
```

Comment:
Can be replaced with:
```
sortBy(key => key.toLowerCase())(Object.keys(recipes));
```

Also it doesn't have to be let, prefer const whenever possible.

Reply:
Done and done. I have also gone through the file and made sure let is used only where necessary.